definitions/benefit/sheets/CaseEventToFields/CaseEventToFields-prod.json

```diff
@@ -18,5 +18,41 @@
     "PageDisplayOrder": 1,
     "PageFieldDisplayOrder": 16,
     "PageID": "1.0"
+  },
```

Comment:
is this supposed to be non-prod, I'm confused?

Reply:
or are we removing that field from the page, so now it will only appear in prod and won't appear in lower environments?